README.md

```diff
@@ -238,13 +283,35 @@ The **summary.tsv** output file generated by `staramr` contains the following co
 | SRR1952908 | aadA1, aadA2, blaTEM-57, cmlA1, gyrA (S83Y), sul3, tet(A) | streptomycin, ampicillin, chloramphenicol, ciprofloxacin I/R, nalidixic acid, sulfisoxazole, tetracycline |
 | SRR1952926 | blaTEM-57, gyrA (S83Y), tet(A)                            | ampicillin, ciprofloxacin I/R, nalidixic acid, tetracycline                                               |
 
+## detailed_summary.tsv
+
+The **detailed_summary.tsv** output file generated by `staramr` contains the following columns:
+
+* __Isolate ID__: The id of the isolate/genome file(s) passed to `staramr`.
+* __Gene__: The particular AMR gene detected from ResFinder, PlasmidFinder, and PointFinder.
+* __Predicted Phenotype__: The predicted AMR phenotype (drug resistances) found in ResFinder.
+* __%Identity__: The % identity of the top BLAST HSP to the AMR gene.
+* __%Overlap__: THe % overlap of the top BLAST HSP to the AMR gene (calculated as __hsp length/total length * 100__).
+* __HSP Length/Total Length__ The top BLAST HSP length over the AMR gene total length (nucleotides).
+* __Contig__: The contig id containing this AMR gene.
+* __Start__: The start of the AMR gene (will be greater than __End__ if on minus strand).
+* __End__: The end of the AMR gene.
+* __Accession__: The accession of the AMR gene from either ResFinder, PlasmidFinder,  database.
+* __Data Type__: The type of gene it is either a **Resistance** gene or a **Plasmid** gene
+
+### Example
+| Isolate ID | Gene        | Predicted Phenotype               | %Identity | %Overlap | HSP Length/Total Length | Contig      | Start  | End    | Accession | Data Type  |
+|------------|-------------|-----------------------------------|-----------|----------|-------------------------|-------------|--------|--------|-----------|------------|
+| SRR1952926 | IncI1       |                                   | 100       | 100      | 142/142                 | contig00017 | 3907   | 3766   | AP005147  | Plasmid    |
+| SRR1952926 | blaTEM-57   | ampicillin                        | 99.88     | 100      | 861/861                 | contig00027 | 6176   | 5316   | FJ405211  | Resistance |
+
 ## resfinder.tsv
 
 The **resfinder.tsv** output file generated by `staramr` contains the following columns:
 
 * __Isolate ID__: The id of the isolate/genome file(s) passed to `staramr`.
 * __Gene__: The particular AMR gene detected.
-* __Predicted Phenotype__: The predicted AMR phenotype (drug resistances) for this gene.
+* __Predicted Phenotype__: The predicted AMR phenotype (drug resistances) for this gene. Plasmid genes will be left blank by default.
```

Comment:
I think you've added this to the wrong section. The sentence **Plasmid genes will be left blank by default** should go into the `detailed_summary` section.

Reply:
Thanks for pointing that out, fixed it in my next commit